Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM ubuntu:18.04
 
 RUN apt-get update\
-     && apt-get install curl -y --no-install-recommends\
```

Comment:
We're currently using curl for healthchecks for local dev in [ras-rm-docker-dev](https://github.com/ONSdigital/ras-rm-docker-dev). We'd need an alternative or to remove it otherwise all our containers will show as permanently unhealthy locally.

Reply:
Would having separate Dockerfiles for local use be an option? I'm really trying to reduce the attack surface of the containers for production.

Reply:
Yes, these images have only been used for dev convenience up till now. I don't know how important it is to us to keep those convenience features in place.

Reply:
I don't see how having curl on a container helps with healthchecks. Curl _makes_ HTTP requests, it doesn't serve HTTP requests. I imagine that `ras-rm-docker-dev` does a curl GET to a `/info` endpoint of the service, to check if it's up. Therefore curl isn't needed, except in the _requester_ which is `ras-rm-docker-dev`.

My guess would be that removing curl from this container will make no difference at all.

Reply:
The health check command is run inside the container, not on the host machine.

Reply:
Looking at it, it's a docker-compose thing where you can configure a healthcheck whereupon the container itself attempts to check the service running inside it (where it uses the curl to hit it's own `/info` endpoint).

Seeing as the healthcheck in this case is used purely in the local dev version (via `rm-ras-docker-dev`) I'd be inclined to remove the `curl`s from here and also remove the healthcheck from the docker-compose in the `docker-dev` repo.